lib/index.ts

```diff
@@ -0,0 +1,6 @@
+export {utf16LeArrayToString, utf8ArrayToString, bigInt64FromParts} from "./utils/utils.js"
```

Comment:
is it correct to import from .js files here?

Reply:
yes, js spec says that the only valid way to import files (as opposed to modules) and TS won't let you import .ts files